Procfile

```diff
@@ -0,0 +1 @@
+web: MIX_ENV=prod mix phoenix.server
```

Comment:
with Phoenix 1.3 can the procfile use ```mix phx.server``` instead of ```phoenix.server``` or are there any blockers for using the new phx command?

Reply:
No, you're right. I was referencing a non-1.3 setup